tailormap-components/projects/core/src/lib/feature-form/form-copy/form-copy.component.css

```diff
@@ -23,6 +23,15 @@
   height: 50%;
 }
 
+.mat-primarylight {
+  background-color: rgba(98, 54, 255, 0.1);
+  color: #6236ff;
+}
+
 .copy-side-panel {
   width: 50%;
 }
+
+.actions button {
```

Comment:
Maybe change this to

```
.actions button + button {
   margin-left: 8px;
}
```

This way you don't have margin on the last button which you dont need